package.json

```diff
@@ -2,17 +2,18 @@
   "name": "open-ai-reviewer",
   "version": "1.0.0",
   "description": "Open AI powered code reviews",
-  "main": "lib/main.js",
+  "main": "lib/index.js",
```

Comment:
The change from `lib/main.js` to `lib/index.js` should be verified to ensure that `index.js` is the correct entry point for the application.